epistemix_platform/src/epistemix_platform/controllers/job_controller.py

```
@@ -14,6 +14,7 @@
 from epistemix_platform.models.job_upload import JobUpload
 from epistemix_platform.models.requests import RunRequest
 from epistemix_platform.models.run import Run
+from epistemix_platform.models.run_results import RunResults
```

Flake8 (7.3.0)

[error] 'epistemix_platform.models.run_results.RunResults' imported but unused

(F401)